src/pybind/mgr/dashboard_v2/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -247,11 +221,15 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
     const val = this.search.toLowerCase();
     const columns = this.columns;
     // update the rows
-    this.rows = this.data.filter(function (d) {
-      return columns.filter((c) => {
-        return (typeof d[c.prop] === 'string' || typeof d[c.prop] === 'number')
-          && (d[c.prop] + '').toLowerCase().indexOf(val) !== -1;
-      }).length > 0;
+    this.rows = this.data.filter(function(d) {
+      return (
+        columns.filter(c => {
+          return (
+            (typeof d[c.prop] === 'string' || typeof d[c.prop] === 'number') &&
```

Comment:
Don't use `typeof` use the lodash functions instead.

Reply:
This was not introduced in this commit, but I will change it.